test/unit/jest.config.json

```diff
@@ -6,7 +6,8 @@
 	},
 	"preset": "@wordpress/jest-preset-default",
 	"setupFiles": [
-		"core-js/fn/symbol/async-iterator"
+		"core-js/fn/symbol/async-iterator",
+		"./test/unit/setup/gutenberg-phase.js"
```

Comment:
I would put do it once using [globalSetup](https://jestjs.io/docs/en/configuration#globalsetup-string). `setupFiles` runs for every test suite file and it might be too late if we ever reference the code which depends on the flag before.

By the way, the safest way to reference a file is to use `<rootDir>/test/unit/setup/gutenberg-phase.js` - which probably is the same in this case :)

Reply:
`cross-env GUTENBERG_PHASE=2 wp-scripts test-unit-js --config test/unit/jest.config.json` didn't work?

Reply:
> `cross-env GUTENBERG_PHASE=2 wp-scripts test-unit-js --config test/unit/jest.config.json` didn't work?

No, not sure why. 

> I would put do it once using [globalSetup](https://jestjs.io/docs/en/configuration#globalsetup-string). `setupFiles` runs for every test suite file and it might be too late if we ever reference the code which depends on the flag before.

Thanks, I'll give that a go, it'll mean switching our config from json to js, assuming that's ok.

Reply:
`globalSetup` is like `setupFiles` but I think it takes string - path to file. So I don't think you need to change the format of config.

Reply:
Oh right. Though I noticed this in the docs:
> Note: Any global variables that are defined through globalSetup can only be read in globalTeardown. You cannot retrieve globals defined here in your test suites.

So it seems as thought it wouldn't be usable for this purpose.

Reply:
Now, I see that. I got confused by the example I saw in the past:

https://jestjs.io/docs/en/puppeteer#custom-example-without-jest-puppeteer-preset

Reply:
> It's also worth noting that `setupFiles` will execute before `setupFilesAfterEnv` (`setupTestFrameworkScriptFile` in our version of Jest).

It should be okey as is then if we can't setup it once for all files. Those might be also two seperate run contexts.